docs/elasticsearch-spec.asciidoc

```diff
@@ -359,64 +359,59 @@ The init container inherits the image of the main container image if one is not
 [id="{p}-update-strategy"]
 === Update strategy
 
-The Elasticsearch cluster configuration can be updated at any time to:
+Users can choose to constraint the operator on the resources it can use when updating using `updateStrategy' like below.
```

Comment:
```suggestion
You can limit the number of simultaneous changes the operator will perform using an `updateStrategy` as follows:
```

Reply:
Would that be specific enough? I'm worried that "changes" is too broad. WDYT?

Reply:
I see what youre saying, but we do clarify what changes we mean shortly there after which might be enough?

Maybe it would make sense to add a conceptual overview before we get into specific examples? Something along the lines of: "sometimes the operator must take a pod down, say to restart a node and apply a new configuration value. Other times an operator must spin up pods above what is currently in the spec, say to migrate to a new nodeSet"